src/schema/schema.graphql

```diff
@@ -2645,9 +2656,9 @@ type Query {
 		types: SearchType
 	): SearchResponse
 	getTasks(
-		query: String!,
-		limit: Int!,
-		offset: Int!,
+		query: String!
```

Comment:
@jkester1986 ignore these changes, I have rebased your PR to remove extra commits and ran this file through prettier